CONTRIBUTING.md

```diff
@@ -143,10 +143,8 @@ Sometimes, we want to release a version we can test before releasing it to the p
 In order to do that, please follow this:
 
 1. Manually change the version in `package.json` and `package-lock.json` to a dev version. Example: `1.4.0-dev1`
-2. Go to publish-please's config in `.publishrc`
-3. Change the `publishTag` to `next` and `gitTag` to `false` (publish-please will expect the latest commit to have a git tag, but we don't want it in this case)
-4. Commit your changes to `trunk`
-5. Run `npm run publish-please`
+1. Commit your changes to `trunk`
```

Comment:
I guess this won't be required?

Reply:
Removed in https://github.com/Automattic/vip-cli/pull/1419/commits/922719888f634db2652d11ec1ad130b3c65a4a7c